qemu/tests/blockdev_commit_powerdown.py

```diff
@@ -0,0 +1,66 @@
+from virttest import env_process
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitHotunplug(BlockDevCommitTest):
+
+    def commit_snapshot_and_destory_vm(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
```

Comment:
we might have to set a small speed in order to keep job running for some time for our check

Reply:
Ok,Thanks.